intake_geopandas/geopandas.py

```diff
@@ -172,10 +178,45 @@ def _open_dataset(self):
                 )
         else:
             self._dataframe = geopandas.read_parquet(
-                self.urlpath,
-                **self._geopandas_kwargs
+                self.urlpath, **self._geopandas_kwargs
             )
 
+    def _get_schema(self):
+        if self._df is None:
+            self._df = self._to_dask()
+        dtypes = {k: str(v) for k, v in self._df._meta.dtypes.items()}
+        self._schema = Schema(
+            datashape=None,
+            dtype=dtypes,
+            shape=(None, len(self._df.columns)),
+            npartitions=self._df.npartitions,
+            extra_metadata={},
+        )
+        return self._schema
+
+    def _load_metadata(self):
+        return super()._load_metadata()
+
+    def to_dask(self):
+        self._load_metadata()
+        return self._df
+
+    def read(self):
+        return self.to_dask().compute()
+
+    def _to_dask(self):
+        """
+        Create a lazy dask-geodataframe from the parquet data
+        """
+        import dask_geopandas
+
+        urlpath = self._get_cache(self.urlpath)[0]
```

Comment:
What is the purpose of this?

Reply:
taken from https://github.com/intake/intake-parquet/blob/master/intake_parquet/source.py#L107

Reply:
This is the "old" caching system for intake. On this line, files could be downloaded, and a local path returned. While the old system continues to be maintained, there's no need to implement it for anything new, since you can now leave any caching to fsspec with compound URLs.